how REVISION

```diff
@@ -0,0 +1,76 @@
+[33mcommit ce194e677659a9f500a7f0affc8957d236918f35[m[33m ([m[1;36mHEAD -> [m[1;32missue_5[m[33m)[m
```

Comment:
I guess this file ended up here by mistake.

Reply:
Not sure by which means it has been created.

Reply:
Thinking, maybe by piping output of  git log into a file, ex.: `git log -3 > REVISION`

Reply:
I

> 
> 
> Thinking, maybe by piping output of git log into a file, ex.: `git log -3 > REVISION`

Then if we don't need this file I'll just remove it.